src/test/java/org/fenixedu/academic/domain/UnitTest.java

```diff
@@ -0,0 +1,62 @@
+package org.fenixedu.academic.domain;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Locale;
+import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
+import org.fenixedu.academic.domain.organizationalStructure.AccountabilityType;
+import org.fenixedu.academic.domain.organizationalStructure.ConnectionRule;
+import org.fenixedu.academic.domain.organizationalStructure.PartyType;
+import org.fenixedu.academic.domain.organizationalStructure.PartyTypeEnum;
+import org.fenixedu.academic.domain.organizationalStructure.Unit;
+import org.fenixedu.commons.i18n.LocalizedString;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.FenixFrameworkRunner;
+
+import pt.ist.fenixframework.FenixFramework;
+
+@RunWith(FenixFrameworkRunner.class)
+public class UnitTest {
```

Comment:
There's already a test class for Units, called OrganizationalStructureTest. I believe this test fits there best.